content/en/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -229,7 +229,7 @@ can happen, according to:
 
 ## Pod disruption conditions {#pod-disruption-conditions}
 
-{{< feature-state for_k8s_version="v1.26" state="beta" >}}
+{{< feature-state for_k8s_version="v1.27" state="beta" >}}
 
 {{< note >}}
 If you are using an older version of Kubernetes than {{< skew currentVersion >}}
```

Comment:
This note looks odd. We don't do this anywhere else. But I'll defer to sig docs.

Reply:
what I mean is that we generally don't document old behavior. Users are supposed to just check the documentation for the older version. Not sure why we have this note.

Reply:
Yeah, the entire note was requested by sig-docs. Anyway, I agree it should be a general rule that users should refer to the documentation version for the kubernetes version that they are using. Removing the note for now.